Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ocaml/opam:alpine as base
+
+RUN opam depext -iy jbuilder
+
+COPY . /home/opam/src/travis-opam
+RUN sudo chown opam.nogroup -R /home/opam/src/travis-opam
+
+WORKDIR /home/opam/src/travis-opam
+
+RUN sed -i "s/^;\(.*static.*\)$/\1/" src/jbuild
+RUN opam config exec -- jbuilder build
+RUN sudo cp /home/opam/src/travis-opam/_build/default/src/ci_opam.exe /usr/bin/ci-opam
+
+WORKDIR /src
+ENTRYPOINT [/usr/bin/ci-opam]
```

Comment:
Can these 3 lines be dropped given that you use a 2-phase build?

Reply:
fixed, thanks